service/controller/v25/resource/tccp/update.go

```diff
@@ -213,16 +205,17 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 			desiredStackState.MasterInstanceResourceName = currentStackState.MasterInstanceResourceName
 			desiredStackState.DockerVolumeResourceName = currentStackState.DockerVolumeResourceName
 
-			updateStackInput, err := r.computeUpdateState(ctx, cr, desiredStackState)
+			templateBody, err := r.newTemplateBody(ctx, cr, desiredStackState)
 			if err != nil {
 				return StackState{}, microerror.Mask(err)
 			}
 
 			updateState := StackState{
-				Name:             desiredStackState.Name,
-				ShouldScale:      true,
-				ShouldUpdate:     false,
-				UpdateStackInput: updateStackInput,
```

Comment:
Can `UpdateStackInput` also be removed from `StackState` struct definition?

Reply:
For sure. There will happen more. The whole struct might disappear when I find a way to use the simple resource interface. One step at a time.

Reply:
I dropped the property.